sdcm/logcollector.py

```diff
@@ -814,6 +813,28 @@ def collect_logs(self, local_search_path=None) -> list[str]:
         return super().collect_logs(local_search_path)
 
 
+def collect_kallsyms_map(node):
+
+    LOGGER.info('Collecting kallsyms map on host: %s', node.name)
+    if remote_node_dir := create_remote_storage_dir(node):
+        kallsyms_dir = os.path.join(node.logdir, KALLSYMS_DIR_NAME)
+        os.makedirs(kallsyms_dir, exist_ok=True)
+        # name = f'kallsyms_{datetime.datetime.now().strftime("%Y%m%d_%H%M%S")}'  # TODO: remove if not needed.
+        kallsyms_name = f'kallsyms_{node.uptime}'
+        kallsyms_file_path = os.path.join(kallsyms_dir, kallsyms_name)
+        if os.path.exists(kallsyms_file_path):
+            LOGGER.debug("The kallsyms file '%s' already exists and not changed. Not collecting it's map",
+                         kallsyms_file_path)
+            return
+        log_entity = CommandLog(name=kallsyms_name,
+                                command='sudo cat /proc/kallsyms')
+
+        try:
+            log_entity.collect(node, kallsyms_dir, remote_node_dir)
```

Comment:
you are not passing here the `kallsyms_file_path`

Reply:
no need.